python/src/Interpolation.python.cpp

```diff
@@ -36,4 +36,4 @@ void wrapInterpolation( python::module& module ) {
   .value( "chargedparticle", Component::chargedparticle );
 }
 
-} // namespace rmatrix
+} // namespace core
```

Comment:
I wonder how this happened. :-D

Reply:
It's called a copy/paste and you know it ;-)